@xen-orchestra/lite/src/views/xoa-deploy/XoaDeployView.vue

```diff
@@ -0,0 +1,677 @@
+<template>
+  <TitleBar :icon="faDownload">{{ $t("deploy-xoa") }}</TitleBar>
+  <div v-if="deploying" class="status">
+    <img src="@/assets/xo.svg" width="300" alt="Xen Orchestra" />
+    <div>
+      <h2 v-if="error !== undefined">{{ $t("xoa-deploy-failed") }}</h2>
+      <h2 v-else-if="url !== undefined">{{ $t("xoa-deploy-successful") }}</h2>
+      <h2 v-else>{{ $t("xoa-deploy") }}</h2>
+
+      <UiIcon
+        v-if="error !== undefined"
+        :icon="faExclamationCircle"
+        class="danger"
+      />
+      <UiIcon
+        v-else-if="url !== undefined"
+        :icon="faCircleCheck"
+        class="success"
+      />
+      <!-- TODO: add progress bar -->
+      <p v-else>{{ status }}</p>
+    </div>
+
+    <div v-if="error !== undefined" class="error">
+      <strong>{{ $t("check-errors") }}</strong>
+      <UiRaw>{{ error }}</UiRaw>
+    </div>
+    <p v-else-if="url === undefined" class="warning">
+      <UiIcon :icon="faExclamationCircle" /> {{ $t("keep-page-open") }}
+    </p>
+
+    <UiButton
+      v-if="error !== undefined"
+      :icon="faDownload"
+      @click="resetValues()"
+    >
+      {{ $t("xoa-deploy-retry") }}
+    </UiButton>
+    <UiButton
+      v-else-if="url !== undefined"
+      :icon="faArrowUpRightFromSquare"
+      @click="openXoa"
+    >
+      {{ $t("access-xoa") }}
+    </UiButton>
+    <UiButton
+      v-else
+      @click="cancel()"
+      :disabled="vmRef === undefined"
+      color="error"
+      outlined
+    >
+      {{ $t("cancel") }}
+    </UiButton>
+  </div>
+
+  <div v-else-if="isMobile" class="not-available">
+    <p>{{ $t("deploy-xoa-available-on-desktop") }}</p>
+  </div>
+
+  <div v-else class="card-view">
+    <UiCard>
+      <form @submit.prevent="deploy">
+        <FormSection :label="$t('configuration')">
+          <div class="row">
+            <FormInputWrapper
+              :label="$t('storage')"
+              :help="$t('gb-required', { n: 20 })"
+            >
+              <FormSelect v-model="srId" required>
+                <option disabled value="">
+                  {{ $t("select.storage") }}
+                </option>
+                <option
+                  v-for="sr in filteredSrs"
+                  :value="sr.uuid"
+                  :key="sr.uuid"
```

Comment:
I'd suggest to use directly the selected SR as `v-model` and set the options value to `sr` + set its default value to `undefined`:

```suggestion
              <FormSelect v-model="selectedSr" required>
                <option disabled :value="undefined">
                  {{ $t("select.storage") }}
                </option>
                <option
                  v-for="sr in filteredSrs"
                  :value="sr"
                  :key="sr.uuid"
```